### PR TITLE
Fix invalid link in README.adoc

### PR DESCRIPTION
**Pull request by 2011eric**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## [3.2.1] - 2022-08-01
```
Comment (jserv) [inline CHANGELOG.md:3]: This message should be updated by maintainers.

Comment (neelgala) [reply CHANGELOG.md:3]: you can add a line saying "Fix invalid link in README.adoc" and then I can merge this

Comment (2011eric) [reply CHANGELOG.md:3]: > you can add a line saying "Fix invalid link in README.adoc" and then I can merge this

Should I write this?
```
# CHANGELOG

Fix invalid link in README.adoc

## [3.2.0] - 2022-07-19
  - Add definition of LREGWU according to XLEN.
  - Fix corresponding fcvt.\*.wu tests.
  ```

**Review by pawks — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (pawks) [reply CHANGELOG.md:3]: The changelog update is correct right now. This can be merged.

**Pull request merged**